src/index.js

```diff
@@ -0,0 +1,32 @@
+import './styles.css';
+
+const list = document.querySelector('#list');
+
+const tasks = [
+  {
+    description: 'wash car',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'call mum',
+    completed: false,
+    index: 3,
+  },
+
+];
+
+tasks.sort((a, b) => a.index - b.index);
+
```

Comment:
- [ ] I have checked your code but am not able to run the app in my locally, see the image below and kindly fix this and resubmit your work
<img width="698" alt="1" src="https://user-images.githubusercontent.com/77843050/221163871-3d80d065-46e5-40a0-86d1-7b8d4b668a3a.png">